packages/autocomplete-core/src/__tests__/getEnvironmentProps.test.ts

```diff
@@ -5,9 +5,93 @@ describe('getEnvironmentProps', () => {
   test.todo('forwards the remaining props');
 
   describe('onTouchStart', () => {
-    test.todo('is a noop when panel is not open');
-    test.todo('is a noop when the event target is the input element');
-    test.todo('blurs input if the target is outside Autocomplete');
+    test('is a noop when panel is not open', () => {
+      const {
+        getEnvironmentProps,
+        inputElement,
+        formElement,
+      } = createPlayground(createAutocomplete, {});
+      const panelElement = document.createElement('div');
+
+      const { onTouchStart } = getEnvironmentProps({
+        inputElement,
+        formElement,
+        panelElement,
+      });
+      window.addEventListener('touchstart', onTouchStart);
+
+      // Focus input (with the panel closed)
+      inputElement.focus();
+
+      // Dispatch TouchStart event on window
+      const customEvent = new CustomEvent('touchstart', {
+        bubbles: true,
+      });
+      window.dispatchEvent(customEvent);
+    });
```

Comment:
There's no assertion in this test.

```suggestion

      expect(document.activeElement).toBe(inputElement);
    });
```

Reply:
Oups :grimacing: